application/api/console.1/content.js

```diff
@@ -1,9 +1,13 @@
 ({
   access: 'public',
+
   async method({ name }) {
+    // Try type: new api.console.content.CustomError('EPARSE');
     const filePath = `/content/${name}.md`;
     const buffer = application.resources.get(filePath);
     if (!buffer) return new Error('Content is not found');
     return { text: buffer.toString() };
   },
+
+  CustomError: class CustomError extends DomainError {},
```

Comment:
Rarely you need an error per-method.
It is much easier to manage errors defined in a separate file and exported everywhere.
This way format/code/message/interface is much easier to control.